ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/builders/JarManifestErrorReporter.java

```diff
@@ -139,17 +141,32 @@ protected void parseManifest(IDocument document, IProgressMonitor monitor) {
 				}
 				// parse
 				if (line.length() == 0) {
-					// Empty Line
+					// Empty Line in the beginning
 					if (l == 0) {
 						report(PDECoreMessages.BundleErrorReporter_noMainSection, 1, CompilerFlags.ERROR, PDEMarkerFactory.CAT_FATAL);
 						return;
+
+						// empty line elsewhere
+					} else if (l != document.getNumberOfLines() - 1) {
+						emptyLines.add(Integer.valueOf(l));
+						continue;
 					}
 					/* flush last line */
 					if (header != null) {
 						fHeaders.put(header.getName().toLowerCase(), header);
 						header = null;
 					}
 					break; /* done processing main attributes */
+				} else {
+					for (Integer index : emptyLines) {
+						VirtualMarker marker = report(PDECoreMessages.BundleErrorReporter_noNameHeader, index + 1,
+								CompilerFlags.WARNING, PDEMarkerFactory.M_EXTRANEOUS_EMPTY_LINES,
+								PDEMarkerFactory.CAT_OTHER);
```

Comment:
`PDEMarkerFactory.CAT_OTHER` is intentional? It is not specified in the error reported above.

Reply:
the other categories are so specific and do not match with this context. If and when we decide to elevate this into an error, we could use `PDEMarkerFactory.CAT_FATAL` .WDYT?

Reply:
Sure. I'm not an OSGI expert, but if you know that manifest containing an empty line is invalid, it is surely a fatal issue?

Reply:
fixed this too - using `PDEMarkerFactory.CAT_FATAL` now.